packages/vue/src/defaults/DefaultTexts.ts

```diff
@@ -42,3 +42,8 @@ export const CODE_TEXT = 'Code';
 export const PASSWORD_TEXT = 'Password';
 export const HIDE_PASSWORD = 'Hide password';
 export const SHOW_PASSWORD = 'Show password';
+export const SIGN_IN_GOOGLE = 'Sign in with Google';
```

Comment:
@wlee221 What's our latest answer for directly referencing our translations (and getting type safety) vs. maintaining an enum lookup?

Reply:
Ideally we should use `DefaultTexts` from `@aws-amplify/ui`: https://github.com/aws-amplify/amplify-ui/blob/36ce52646f455e6a3f2d7e9befb4f808efc1061b/packages/ui/src/i18n/translations.ts#L27-L35

This will be the source of truth across the frameworks.

Reply:
If you wish to use to use the string literals directly, you can do so with `translate` helper I provided here: https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/i18n/translations.ts#L82-L84. This type is inferred from `DefaultTexts` above, so this is consistent with the above enum.

Reply:
@wlee221 @ericclemmons Perhaps in a separate pull request I can start pulling the values from the ui translations file. The defaultTexts is just the source of truth for all Vue.js texts. But it would be nice to use the same one used in Angular and React.